pandas/core/generic.py

```diff
@@ -6098,9 +6098,9 @@ def replace(self, to_replace=None, value=None, inplace=False, limit=None,
             * 'index', 'values': use the actual numerical values of the index
             * 'nearest', 'zero', 'slinear', 'quadratic', 'cubic',
               'barycentric', 'polynomial' is passed to
-              ``scipy.interpolate.interp1d``. Both 'polynomial' and 'spline'
-              require that you also specify an `order` (int),
-              e.g. df.interpolate(method='polynomial', order=4).
+              :func:`scipy.interpolate.interp1d`. Both 'polynomial' and
```

Comment:
Seems like `scipy.interpolate.interp1d` is a class. May be that's the problem, as it requires `:class:` instead of `:func:`?

https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.interp1d.html